### ISSUE TITLE
[BETA]: Logo image does not have visible focus ring

### ISSUE THREAD

**Issue by aaronjordan** [aaronjordan/aaronjordan-me-v3#5]
When focused, the site title should clearly indicate its state, but doesn't.

**Comment by aaronjordan** [comment]
Fixed in latest revision.
